src/components/Homepage/Hero/Hero.jsx

```diff
@@ -36,7 +36,8 @@ const Hero = () => {
 
                 <HeroBtnWrapper>
                     <ScrollButton
-                        to={"about"}
+                        href="/about"
```

Comment:
what is the need of href here?

we have already have a to=""

Reply:
Links are not crawlable if they do not have href

refer to this: https://developers.google.com/search/docs/crawling-indexing/links-crawlable?visit_id=638433557056797392-504149247&rd=1

without href attribute it will be like this:
![Screenshot 2024-02-12 093024](https://github.com/thecyberworld/TheCyberHUB/assets/79264045/1b75feee-e257-40bc-b4c8-74a5269d2e13)